packages/wrangler/src/triggers/deploy.ts

```diff
@@ -315,7 +317,8 @@ async function subdomainDeploy(
 	envName: string,
 	workerUrl: string,
 	routes: Route[],
-	deployments: Array<Promise<string[]>>
+	deployments: Array<Promise<string[]>>,
+	forceSubdomainDeploy: boolean
```

Comment:
Do we really need to an arg, given that we pass `props.forceSubdomainDeploy` and props is already the first arg?

Reply:
I feel like it is good to be explicit about forcing here. I want to get rid of `props` arg eventually.